hackeriet/web/httpbridge/__init__.py

```diff
@@ -38,50 +44,56 @@ def add_header(response):
     response.cache_control.max_age = 0
     return response
 
+# these gifs are just 1x1 pictures of nothing
+# what is the point of this?
 @app.route("/hackeriet.gif")
 def gif():
     n = random.choice([1,2,3,4,5,6])
     if humla == "OPEN":
-        return redirect("/static/img/open/" + str(n) + ".gif")
+        return redirect(f"/static/img/open/{n}.gif")
     else:
-        return redirect("/static/img/closed/" + str(n) + ".gif")
+        return redirect(f"/static/img/closed/{n}.gif")
 
 @app.route("/humla")
 def tut():
     return humla
 
 @app.route("/topic.jsonp")
 def topictut():
-    t = "hackerietTopic(\"" + topic + "\")"
+    t = f'hackerietTopic("{topic}")'
     return t, 200, {'Content-Type': 'application/javascript'}
 
 @app.route("/door.json")
 def door_json():
-    t = "{ \"status\": \"%s\" }" % humla
+    body = {"status":humla.lower(), "is_open":humla=="OPEN"}
+    t = json.dumps(body)
     return t, 200, {'Content-Type': 'application/javascript'}
 
 @app.route("/", methods=["GET", "POST"])
 def hello():
     addr = request.remote_addr
-    if addr == "::1" or addr == "localhost" or addr == "127.0.0.1" and 'X-Forwarded-For' in request.headers:
-        addr = request.headers['X-Forwarded-For']
 
-    if request.method == 'POST':
-        if request.form['person']:
-            person = request.form['person']
-        else:
-            person = ''
+    if addr in ("::1", "localhost", "127.0.0.1") and 'X-Forwarded-For' in request.headers:
```

Comment:
This can be configured globally as well - https://flask.palletsprojects.com/en/stable/deploying/proxy_fix/